products/ffmpeg.md

```diff
@@ -3,235 +3,251 @@ title: FFmpeg
 category: framework
 iconSlug: ffmpeg
 permalink: /ffmpeg
+versionCommand: ffmpeg -version
 releasePolicyLink: https://ffmpeg.org/
 changelogTemplate: https://ffmpeg.org/download.html#release_{{"__RELEASE_CYCLE__"}}
-versionCommand: ffmpeg -version
+releaseLabel: "__RELEASE_CYCLE__ '__CODENAME__'"
 activeSupportColumn: false
 releaseDateColumn: true
 eolColumn: Supported
+
 auto:
-  # upstream https://git.ffmpeg.org/ffmpeg.git doesn't support filtering
+# upstream https://git.ffmpeg.org/ffmpeg.git doesn't support filtering
 -   git: https://github.com/FFmpeg/FFmpeg.git
     regex: '^n?(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.?(?<patch>0|[1-9]\d*)?$'
-releaseLabel: "__RELEASE_CYCLE__ '__CODENAME__'"
+
 releases:
+-   releaseCycle: "6.0"
+    codename: Von Neumann
+    lts: true
+    releaseDate: 2023-02-27
+    eol: false
+    latest: "6.0.0"
+    latestReleaseDate: 2023-02-27
+
 -   releaseCycle: "5.1"
     codename: Riemann
+    lts: true
+    releaseDate: 2022-07-22
     eol: false
     latest: "5.1.2"
-    lts: true
     latestReleaseDate: 2022-09-25
-    releaseDate: 2022-07-22
+
 -   releaseCycle: "5.0"
     codename: Lorentz
+    lts: true
+    releaseDate: 2022-01-14
     eol: false
     latest: "5.0.2"
-    lts: true
     latestReleaseDate: 2022-11-04
-    releaseDate: 2022-01-14
 
 -   releaseCycle: "4.4"
     codename: Rao
+    releaseDate: 2021-04-08
     eol: false
     latest: "4.4.3"
     latestReleaseDate: 2022-10-09
-    releaseDate: 2021-04-08
 
 -   releaseCycle: "4.3"
     codename: '4:3'
+    releaseDate: 2020-06-15
     eol: false
     latest: "4.3.5"
     latestReleaseDate: 2022-10-10
-    releaseDate: 2020-06-15
 
 -   releaseCycle: "4.2"
     codename: 'Ada'
+    releaseDate: 2019-08-05
     eol: false
     latest: "4.2.8"
     latestReleaseDate: 2022-10-11
-    releaseDate: 2019-08-05
 
 -   releaseCycle: "4.1"
     codename: 'al-Khwarizmi'
+    releaseDate: 2018-11-06
     eol: false
     latest: "4.1.10"
     latestReleaseDate: 2022-10-21
-    releaseDate: 2018-11-06
 
 -   releaseCycle: "4.0"
     codename: 'Wu'
+    releaseDate: 2018-04-20
     eol: 2020-07-03
     latest: "4.0.6"
     latestReleaseDate: 2020-07-03
-    releaseDate: 2018-04-20
 
 -   releaseCycle: "3.4"
     codename: 'Cantor'
+    releaseDate: 2017-10-15
     eol: false
     latest: "3.4.12"
     latestReleaseDate: 2022-10-27
-    releaseDate: 2017-10-15
 
 -   releaseCycle: "3.3"
     codename: 'Hilbert'
+    releaseDate: 2017-04-13
     eol: 2018-11-18
     latest: "3.3.9"
     latestReleaseDate: 2018-11-18
-    releaseDate: 2017-04-13
 
 -   releaseCycle: "3.2"
     codename: 'Hypatia'
+    releaseDate: 2016-10-27
     eol: false
```

Comment:
3.2 is on [old releases](https://ffmpeg.org/olddownload.html) and thus eol on `latestReleaseDate`
```suggestion
    eol: 2022-10-28
```